src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -126,20 +128,20 @@ def setUp(self):
         self.daos_cmd = self.get_daos_command()
 
         # Get the processes and np for all datamover tools, as well as for individual tools.
-        self.processes = self.params.get("np", '/run/datamover/*', 1)
-        self.ppn = self.params.get("ppn", '/run/datamover/*', 1)
+        self.datamover_np = self.params.get("np", '/run/datamover/*', 1)
+        self.datamover_ppn = self.params.get("ppn", '/run/datamover/*', 1)
```

Comment:
Is this necessary? I started this pattern, and it's pretty messy (my fault). But I would think any tests using the datamoer test base should just use `self.ior_np`, etc. as needed, since the previous `self.processes = self.params.get("np", '/run/datamover/*', 1)` overrides it.

Reply:
If this solves a problem, I'm okay to keep it. I'll eventually need to refactor this anyway, since I started a mess :)

Reply:
Indeed, there is probably cleaner ways to solve this issue, however it should require in depth modifications of the code.
I have to admit, that I would prefer to avoid making such big code refactorings with this PR as it is not aimed for that.

Reply:
Agree. No need for you to do more work